src/pages/contact-us/_offices.js

```diff
@@ -210,24 +248,28 @@ export const Offices = () => {
                                 target="_blank"
                                 href="https://g.page/r/CQODFgzIJPYtEAE"
                             >
-                                <Image
-                                    img_name="map-cyberjaya.png"
-                                    alt={localize('Malaysia Office')}
-                                    width="49rem"
-                                    height="100%"
-                                />
+                                <MapContainer>
+                                    <Image
```

Comment:
possible to replace the `<Image` with `QueryImage`? 🙏

Reply:
sure